src/BulletSoftBody/btDeformableContactConstraint.h

```diff
@@ -154,6 +181,13 @@ class btDeformableRigidContactConstraint : public btDeformableContactConstraint
     virtual btVector3 getVa() const;
     
     virtual btScalar solveConstraint();
+    
+    virtual btScalar solveSplitImpulse(const btContactSolverInfo& infoGlobal);
+    
+    virtual void setPenetrationScale(btScalar scale)
```

Comment:
Is the 'scale' the amount of penetration to be resolved? In other parts of the code, we call this the 'error reduction parameter'.

Reply:
Yes, this is the same as erp. I will modify.